<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>747</width>
        <height>1024</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="Domain_lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>id service</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Domain_lineEdit_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>id user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Domain_lineEdit_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>name user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Domain_lineEdit_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>name service</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>نوع الخدمة</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="customer_comboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Domain</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WebHosting</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>mobaileApps</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WebApps</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>DesktopApps</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="Domain_lineEdit_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maxLength">
                   <number>50</number>
                  </property>
                  <property name="placeholderText">
                   <string>اسم الموقع</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QFrame" name="frame_8">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>بداية التاريخ</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="Domain_dateEdit_4">
                     <property name="toolTip">
                      <string>start date</string>
                     </property>
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="wrapping">
                      <bool>false</bool>
                     </property>
                     <property name="frame">
                      <bool>true</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="keyboardTracking">
                      <bool>true</bool>
                     </property>
                     <property name="dateTime">
                      <datetime>
                       <hour>0</hour>
                       <minute>0</minute>
                       <second>0</second>
                       <year>2019</year>
                       <month>10</month>
                       <day>1</day>
                      </datetime>
                     </property>
                     <property name="currentSection">
                      <enum>QDateTimeEdit::YearSection</enum>
                     </property>
                     <property name="displayFormat">
                      <string>dd-MM-yyyy</string>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>نهاية التاريخ</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="Domain_dateEdit_5">
                     <property name="toolTip">
                      <string>end date</string>
                     </property>
                     <property name="frame">
                      <bool>true</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::UpDownArrows</enum>
                     </property>
                     <property name="accelerated">
                      <bool>false</bool>
                     </property>
                     <property name="correctionMode">
                      <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                     </property>
                     <property name="dateTime">
                      <datetime>
                       <hour>0</hour>
                       <minute>0</minute>
                       <second>0</second>
                       <year>2019</year>
                       <month>10</month>
                       <day>1</day>
                      </datetime>
                     </property>
                     <property name="currentSection">
                      <enum>QDateTimeEdit::YearSection</enum>
                     </property>
                     <property name="displayFormat">
                      <string>dd-MM-yyyy</string>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_10">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_32">
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="Domain_textEdit">
               <property name="documentTitle">
                <string/>
               </property>
               <property name="overwriteMode">
                <bool>false</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextEditorInteraction</set>
               </property>
               <property name="placeholderText">
                <string>البيانات | هذا الحقل يستم تشفيرة لحفظ السرية</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_49">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_34">
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_50">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string> التكلفة</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Domain_lineEdit_23">
               <property name="placeholderText">
                <string>سعر التكلفة</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>البيع</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Domain_lineEdit_24">
               <property name="placeholderText">
                <string>سعر البيع</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Domain_pushButton">
            <property name="text">
             <string>إضافة</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Domain_pushButton_Edite">
            <property name="text">
             <string>حفظ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
